packages/ecs-agent/Cargo.toml

```diff
@@ -13,8 +13,8 @@ path = "../packages.rs"
 releases-url = "https://github.com/aws/amazon-ecs-agent/releases"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/aws/amazon-ecs-agent/archive/v1.75.0/amazon-ecs-agent-1.75.0.tar.gz"
```

Comment:
There is a newer version for the ECS agent.

Reply:
I believe that is yet to [release](https://github.com/aws/amazon-ecs-agent/releases)